doc/api/tty.md

```diff
@@ -88,6 +88,10 @@ process.stdout.on('resize', () => {
 });
 ```
 
+*Note:* Terminal window resize events are unreliable on Windows. The event 
+handler is only executed for changes in the width of the window and the 
```

Comment:
If the console buffer height is changed, the event will also triggered. The buffer is not shrinked when downsizing widow, so it can look like changing height is ignored. If however one sets buffer height to something small in the options and then increase window height the event will be triggered.